src/context.js

```diff
@@ -30,28 +34,20 @@ const CONTEXTS = 'contexts'
 const CONFIG = 'config'
 
 /** Property holding the cli context name */
-const CLI = 'cli'
+const CLI = process.env.AIO_CLI_CONTEXT || 'cli'
```

Comment:
We should document this in Readme.

Reply:
This pr is somewhat less important than the other changes.  This was done before when we were worried about storing multiple contexts.  I think if wxp is okay with this config key/path (in their own global config file) then we can just close this PR.

```
  contexts: {
    cli: {
    ...

```